packages/vite/src/migrations/update-15-3-1/update-vite-tsconfig-paths.ts

```diff
@@ -7,6 +7,7 @@ import {
 } from '@nrwl/devkit';
 
 import { forEachExecutorOptions } from '@nrwl/workspace/src/utilities/executor-options-utils';
+import { tsquery } from '@phenomnomnominal/tsquery';
```

Comment:
Should be `15-4-0` in the file path (minor).

Also needs to be added to `migrations.json` file.

Reply:
Ohhh ok!